src/chttpd/test/eunit/chttpd_db_test.erl

```diff
@@ -209,6 +211,49 @@ should_return_200_for_del_att_with_rev(Url) ->
     end)}.
 
 
+should_not_send_csp_header_with_att_by_default(Url) ->
```

Comment:
since this is now bool-flipped from the 3.x default, we should have a another test that proves that CSP is sent when enabled.

Reply:
Good idea, I"ll add another test for it

Reply:
Added the `should_send_csp_header_with_att_when_configured` test